unit_tests/test_exceptions.py

```diff
@@ -62,6 +62,39 @@ def test_hit_w_content_as_str(self):
         self.assertEqual(exception.message, 'Not Found')
         self.assertEqual(list(exception.errors), [])
 
+    def test_hit_w_content_as_unicode(self):
+        import six
+        from google.cloud.exceptions import NotFound
+        if six.PY3:  # pragma: NO COVER
+            _ERROR_MESSAGE = u'That\u2019s not found.'
+        elif six.PY2:  # pragma: NO COVER
+            _ERROR_MESSAGE = 'Thats not found.'
```

Comment:
You've traded a pragma in source for a pragma in test. If you use the UTF-8 encoding, this could be avoided.